week01/main_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "testing"
+
+func TestAdd(test *testing.T) {
+	test_case := []struct {
+		a    int
+		b    int
+		want int
+	}{
+		{1, 2, 3},
+		{-10, -5, -15},
+		{0, 0, 0},
+	}
+	for _, tc := range test_case {
+		result := add(tc.a, tc.b)
+		if result != tc.want {
+			test.Errorf("Adding(%d,%d) = %d, want %d", tc.a, tc.b, result, tc.want)
+		}
+	}
+}
```

Comment:
@0m-a-D Update commit message into present tense rather than past tense.
Checkout this nice article for some more info https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/